keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -993,6 +923,11 @@ resources were not requested.
     - kube-controller-manager
     - kube-scheduler
     - kubelet
+  - Feature gate name: DRAControlPlaneController
```

Comment:
So, we will error out if this is enabled and the other is not?

Reply:
We can make that an error in the individual components. I don't think we have a way to do it in the feature gate package.